x-pack/filebeat/module/activemq/log/config/log.yml

```diff
@@ -0,0 +1,10 @@
+type: log
+paths:
+{{ range $i, $path := .paths }}
+ - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+multiline:
+  pattern: '^\d{4}-\d{2}-\d{2} '
+  negate: true
+  match: after
```

Comment:
In filesets where we do timezone conversion we add the local timezone by default in `event.timezone` field, for that add the processor in the config:
```
processors:
  - add_locale: ~
```

You will need to regenerate the expected events after this change.

Reply:
Thanks! Fixed and regenerated events after all changes.